src/libraries/apicompat/ApiCompat.proj

```diff
@@ -44,7 +44,7 @@
 
     <!-- Restore and reference assemblies not built in runtime anymore, required for resolving type forward on both comparison sides. -->
     <ApiCompatTypeForwardDestinationPackage Include="System.Data.SqlClient" PackageVersion="$(SystemDataSqlClientVersion)" TargetFramework="netcoreapp2.1" />
-    <ApiCompatTypeForwardDestinationPackage Include="System.Drawing.Common" PackageVersion="$(SystemDrawingCommonVersion)" TargetFramework="net7.0" />
```

Comment:
Was System.Drawing giving you failures that you were not seeing in the main PR?

Reply:
The main PR has the same change. We update System.Drawing.Common to the 8.0.0 version for consistency.